providers/implementations/encode_decode/decode_der2key.c

```diff
@@ -754,7 +760,7 @@ ml_kem_d2i_PUBKEY(const uint8_t **der, long der_len,
         goto end;
     }
 
-    ret = ossl_ml_kem_key_new(libctx, ctx->propq, ctx->desc->evp_type);
+    ret = ossl_ml_kem_key_new(libctx, ctx->propq, 1, ctx->desc->evp_type);
```

Comment:
not that it matters - but why does this set retain_seed =  1 ?

Reply:
It does not matter, yes, it could equally be 0.